arbnode/batch_poster.go

```diff
@@ -66,6 +66,14 @@ var (
 	blobGasLimitGauge             = metrics.NewRegisteredGauge("arb/batchposter/blobgas/limit", nil)
 	suggestedTipCapGauge          = metrics.NewRegisteredGauge("arb/batchposter/suggestedtipcap", nil)
 
+	batchPosterBacklogGauge = metrics.NewRegisteredGauge("arb/batchposter/backlog", nil)
+
+	batchPosterDALastSuccessfulActionGauge = metrics.NewRegisteredGauge("arb/batchPoster/action/da_last_success", nil)
```

Comment:
I think adding a last successful batch posting metric would also make sense to complement this